kustomize/commands/edit/add/configmap_test.go

```diff
@@ -4,13 +4,19 @@
 package add
 
 import (
+	"fmt"
 	"testing"
 
+	"github.com/stretchr/testify/require"
+	"sigs.k8s.io/kustomize/api/provider"
+	"sigs.k8s.io/kustomize/kustomize/v5/commands/internal/kustfile"
+
```

Comment:
nit: remove the newline

Reply:
`goimports` seems to like the newlines 😅

Reply:
Removed!